### DISCUSSION TITLE
Egui supports players and audio playback?

### DISCUSSION THREAD

**Discussion by yuenxillar** [emilk/egui#4971] [Q&A]
Egui supports players and audio playback?l need

**Comment by YgorSouza** [comment]
Egui doesn't handle audio, but you can use [cpal](https://crates.io/crates/cpal) or [rodio](https://crates.io/crates/rodio) and call them from your egui app. It is listed under [companion crates](https://github.com/emilk/egui/tree/1f6ae49a5f6bf43a869c215dea0d3028be8d742a/crates/eframe#companion-crates) in the eframe readme.

**Reply by yuenxillar** [reply]
ok, rodio l know it, but video player Can't find a fit
